src/libraries/System.Private.CoreLib/src/System/Text/Utf8Span.cs

```diff
@@ -204,7 +220,7 @@ public bool IsNormalized(NormalizationForm normalizationForm = NormalizationForm
 
         /// <summary>
         /// Gets an immutable reference that can be used in a <see langword="fixed"/> statement. Unlike
-        /// <see cref="Utf8String"/>, the resulting reference is not guaranteed to be null-terminated.
+        /// TODO cref="Utf8String"/>, the resulting reference is not guaranteed to be null-terminated.
```

Comment:
This looks like a mistake that shouldn't compile. Does it? If it does, we're probably missing enabling XML comment compilation/validation.

Reply:
This was a leftover of getting code building in layers. I've reverted the change.

Just an FYI - in a default .csproj:

```XML
  <PropertyGroup>
    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
    <GenerateDocumentationFile>true</GenerateDocumentationFile>
    <WarningsAsErrors>true</WarningsAsErrors>
  </PropertyGroup>
```

The following code builds without an error or warning:

```c#
    /// <summary>
    /// This is a great class
    /// TODO cref="Utf8String"/>, the resulting reference
    /// </summary>
    public static class Utf8StringProcessor {}
```

But the following gives a "badly formed XML" warning:

```C#
    /// <summary>
    /// This is a great class
    /// <junktag
    /// TODO cref="Utf8String"/>, the resulting reference
    /// </summary>
    public static class Utf8StringProcessor {}
```

So I think the above is "valid" xml doc.